sync/clients/databricks.py

```diff
@@ -48,6 +48,21 @@ def delete_cluster(self, cluster_id: str) -> dict:
             )
         )
 
+    def get_cluster_events(self, cluster_id: str, **kwargs) -> dict:
```

Comment:
Using `**kwargs` here to allow passing arbitrary query params for e.g. paging requests. I didn't want to be too "strict" since Databricks can always decide to send back any random query params for the `next_page` that they'd like to

Reply:
👍 "params" might be more descriptive than "kwargs". I was thinking the optional params might be better encapsulated in a dict like elsewhere in this module, but the POST params make up all the method args so not so important.

- just my 2 cents, no expectation about further changes to this PR